includes/theme-help.php

```diff
@@ -718,58 +433,37 @@
+						To create your slides, click the "Create New Slides" button at the bottom of the window. The Media Library window will close, and any image with a checkbox next to it will be inserted as a slide. New slides are added at the bottom of the list of existing slides.
 					</p>
 
-					<h4 id="photo-essays-slide-content">Title and Caption</h4>
-					<img src="<?php echo THEME_HELP_IMG_URL?>/photo_essay-title-caption.jpg" />
+					<h4 id="photo-essays-slide-content">Caption</h4>
 					<p>
-						When a slide is created from an image in the Media Library, whatever title and caption were set for that image are pulled
-						automatically as the default title and caption for that slide.  To override these without modifying the title/caption of the
-						image in the Media Library, just change the Title and Caption fields for the slide after it's been created.
+						Captions can be added for each image and will appear either below or to the right of the image depending on the image's orientation.
 					</p>
 					<p>
-						Captions modified within the slide can use a mini WYSIWYG editor to apply bold, italics, or underline to text, and also add links.
-					</p>
-					<p>
-						Keep in mind that anything saved here will not modify the corresponding image title/caption in the Media Library.  Modify the image
-						from the Media Library instead to change those values.
+						Keep in mind that setting the caption here will not modify the corresponding image caption in the Media Library.
 					</p>
 
 					<div class="well">
 						<h4 id="photo-essays-requirements">Requirements</h4>
 						<ul>
 							<li>
-								<strong>Each photo essay slide requires a title and image.</strong>  If both of these are not set for any slide, that
-								slide will not be saved when the photo essay is saved/published.
+								<strong>Each photo essay slide requires a caption and image.</strong> If both of these are not set for any slide, that slide will not be saved when the photo essay is saved/published.
 							</li>
 							<li>
-								All images uploaded to photo essays should be <strong>web-optimized</strong>!  Please see the Media Specs/Guidelines
-								section of the <a href="#stories">Story documentation</a> for basic image optimization requirements.
+								All images uploaded to photo essays should be <strong>web-optimized</strong>! Please see the Media Specs/Guidelines section of the <a href="#stories">Story documentation</a> for basic image optimization requirements.
 							</li>
 							<li>
-								<strong>All images uploaded to a single photo essay must be the same height</strong>.  Widths can vary, but the height
-								must be consistent throughout all images.
+								<strong>All images uploaded to a single photo essay must be the same height</strong>. Widths can vary, but the height must be consistent throughout all images.
```

Comment:
This statement can be removed--height only really mattered for slideshows, which we've removed in v6